common/model/HazelcastEntityComparator.js

```diff
@@ -0,0 +1,37 @@
+const HazelcastPortable = require(`./HazelcastPortable`);
+
+
+/**
+ * HazelcastEntityComparator class
+ */
+class HazelcastEntityComparator extends HazelcastPortable {
+
+    static get FACTORY_ID() { return 1; }
+    static get CLASS_ID() { return 7; }
+
+    static getClassName() { return HazelcastEntityComparator.name };
```

Comment:
Semi-colon is missing in the end of return instruction but present in the end of method declaration